src/core/client/stream/tabs/Comments/Stream/StreamContainer.tsx

```diff
@@ -265,7 +272,10 @@ export const StreamContainer: FunctionComponent<Props> = (props) => {
         {isRatingsAndReviews && <StoryRatingContainer story={props.story} />}
         {showCommentForm &&
           (alternateOldestViewEnabled ? (
-            <AddACommentButton isQA={isQA} />
+            <AddACommentButton
+              isQA={isQA}
+              currentScrollRef={currentScrollRef}
```

Comment:
Thanks for keeping this `currentScrollRef` passed through cleanly. It's unfortunate it has to be plumbed through everywhere, but I appreciate that you kept the plumbing down to one variable instead of 3-10 and stored the rest in `Local`. 👍